pytket/pytket/circuit/__init__.py

```diff
@@ -135,7 +135,7 @@ def overload_add_wasm_to_reg(
     if args_wasm is None:
         args_wasm = [0]
 
-    if filehandler._check_file:
+    if filehandler.checked:
```

Comment:
I guess this might be semantically different and therefore not correct as a change